### ISSUE TITLE
Desync issue after using telepads

### ISSUE THREAD

**Issue by DatrixTHLK** [ArtixAllMighty/Telepads2016#2]
It causes a desync issue to occur after using the teleporters to go from one place to another. When I attack mobs before using the teleporters what they drop can be seen when looking into my inventory. After using them, I can't see anything gotten from mobs in the inventory screen unless the character opens a chest. The items will appear back in the inventory.

FML Log: https://gist.github.com/DatrixTHLK/e5aa95b44f6d636aa3f56a06c5bfa505

No crashing involved, but major sync issues occur when trying to open lootbags and birds nests and looking into the characters own inventory to see nothing there until a chest is opened.

**Comment by AbsolemJackdaw** [comment]
This issue is being looked into by me and the original author of the teleportation code, masa.
The origins of this desync issue is most likely due to a vanilla bug, the same one that causes the ever so damning stuck-in-bed glitch after coming from the nether. (aka teleporting)

https://bugs.mojang.com/browse/MC-92216

more about my conversation with masa here : 
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1292377-telepads-taking-teleports-to-a-whole-new-level?comment=422

**Comment by DatrixTHLK** [comment]
I've never had that issue with the stuck in a bed glitch after coming back from the nether. Even without this mod. Anyways, do let me know if anything pops up? I still like this mod.

**Comment by AbsolemJackdaw** [comment]
Me neither. Heard about it trough other people. Just don't worry, we're on it =) 
will let you know when we find a fix.

**Comment by DatrixTHLK** [comment]
Sounds good. I will wait on your word...Senpai. =)

**Comment by DatrixTHLK** [comment]
Issue still exists in the newest version of telepads. Oh well...

**Comment by AbsolemJackdaw** [comment]
agh, id didn't sovle it ! 
I tried it on my client, and it worked great though ...
what do you do to make it desync ? just teleport? or teleport far away ?

**Comment by DatrixTHLK** [comment]
Well I made a teleporter from my house to an underground cave/dungeon not that far underneath where my house is located. x=2 y=72 z=257 to x=-3 y=26 z=230 Technically it is farther underneath my house, but not that far. The only problem is when I use other mods such as Loot bags, Birds Nests and Dungeon Tactics the loot bags don't open right away and have to be placed inside a chest and then out again from them individually to work again. I still can't see the mob drops from the mobs I kill after teleporting. There is also another issue I seem to be experiencing. I am getting a dupe glitch of the armor that I wear on my body in my inventory. When I reboot my game the armor isn't on me but in my inventory.

I am able to teleport to the nether now using a transmitter the problem is that after teleporting to and from the nether the teleporter doesn't show a gui screen anymore with the other points leading back to the overworld. I only teleported three times once from the nether to the over world and back. Something tells me the chunks don't load up when they're suppose to.

**EDIT:** (Zombie Awareness mod issue)
Recent crash report after going to the nether: https://gist.github.com/DatrixTHLK/0187d94b9b8cee0e60e8496fcf2010b6

I have this nagging feeling that the nether went corrupt after going to it and back to the overworld which happened before. The only way I fixed the issue was making a new world. It could also be part of the Desync issue, but who knows. That was only temporary and I don't really feel like making a new world. I'm guessing this is another bug with Minecraft itself that should've been fixed.

Image of nether where chunks fail to load: http://tinypic.com/r/331du3a/9 

**EDIT:** This is also put up in the minecraft forums [here](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/mods-discussion/2758938-failed-chunk-loading).

**EDIT 2:** I'm testing my mod list searching for which one could be causing the corruption in the nether with the no chunks loading/crashing. One problem was most likely Mystical Crops since that mod is unstable.

**Comment by DatrixTHLK** [comment]
Still having sync issue problem. I plant any crops after teleporting to a location with a farm and they vanish after planting them. They will return to my inventory after teleporting to that location again. Won't show up in my inventory as it will be invisble until after I teleport again.

**Comment by DatrixTHLK** [comment]
Actually, I figured out the problem. It was rather obvious from the spam I got in the FML Log that it was Zombie Awareness that was causing the major issues (CPU Usage mainly). This doesn't mean the other issue is fixed. I'm going to further test out if the game still has the issues above minus the CPU Usage.

**Comment by AbsolemJackdaw** [comment]
I'l still trying to find this out. this is so frustrating as I can't find anything... i wish i could blame vanilla, but i'm hoping on fixing vanilla's issue.

**Comment by DatrixTHLK** [comment]
The issue is more than likely fixed, but I'm still trying to test it out in game. I'll get back to you in a few minutes.

Edit: The issue still happens, however it fixes itself once I look into a chest or teleport again. That is all that I can hope for it isn't that major anymore. I'll overlook this minor issue. Thanks again for at least responding to me and trying to figure out where the problem might lie. Always appreciated. :) I'll close the issue.
